_data/current.yml

```diff
@@ -170,6 +170,12 @@
   twitter: PNSQC
   status: CFP is closed
 
+- name: Test Dive Conference 2017
```

Comment:
This whole section should be placed below Tricentis Accelerate 2017 and above TestBash Manchester 2017